services/web-app/components/mdx-notification/mdx-notification.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { ActionableNotification, Button, Column, Grid } from '@carbon/react'
+import { ChevronDown, ChevronUp } from '@carbon/react/icons'
+import clsx from 'clsx'
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+
+import styles from './mdx-notification.module.scss'
+
+export const MdxNotification = ({
+  content,
+  title,
+  description,
+  href,
+  link,
+  id,
+  kind = 'error',
+  collapsible = false
+}) => {
+  const wrapperClassName = clsx({
+    'cds--inline-notification': true
+  })
+
+  const [collapsed, setIsCollapsed] = useState(false)
+
+  return (
+    <Grid
+      narrow
+      className={clsx(
+        !link && styles['no-link'],
+        collapsible && !link && styles['content-container'],
+        styles['error-container']
+      )}
```

Comment:
Do we want to pull this out into a variable to make it easier to read, similar to how we do on other components? Something like 

```
 const notificationClassNames = clsx(styles['error-container'], {
    [styles['no-link']]: !link,
    [styles['content-container']]: collapsible && !link
  })
```

```suggestion
      className={notificationClassNames}
```